src/Form/Form.module.css

```diff
@@ -20,3 +22,30 @@
 .inputLabel {
   padding: 0 1rem;
 }
+
+.radioContainer {
+  display: flex;
+  justify-content: space-around;
+  margin-bottom: 3rem;
+}
+
+.soonColor {
+  color: red;
+}
+
+.kindOfSoonColor {
```

Comment:
I tend to avoid using naming conventions in CSS that describe the style you are applying with the class. The reason is if someone comes along later on and changes this to not apply color, it will be really confusing to the next developer who sees `Color` in the name but only `text-decoration` css property 😄

Reply:
Good call. Updated!